tests/src/test/java/tachyon/client/FileInStreamIntegrationTest.java

```diff
@@ -56,6 +57,9 @@
   private static CreateFileOptions sWriteUnderStore;
 
   @Rule
+  public Timeout mGlobalTimeout = Timeout.seconds(60);
```

Comment:
how long should it last typically? How about on other "slower" profiles like hdfs?